sorc/IMSaggregate_mod.f90

```diff
@@ -336,10 +354,22 @@ subroutine write_IMS_outputs_vec(idim, jdim, otype, date_str,scfIMS, sndIMS, lon
     allocate(data_vec(2,nobs)) 
     allocate(coor_vec(3,nobs)) 
 
-    !--- define dimension
+    !--- define location dimension
     error = nf90_def_dim(ncid, 'numobs', nobs, id_obs)
     call netcdf_err(error, 'defining obs dimension' )
+
+    ! --- define time dimension
+    error = nf90_def_dim(ncid, 'time', nf90_unlimited, dim_time)
+    call netcdf_err(error, 'defining time dimension' )
  
+    !--- define time
+    error = nf90_def_var(ncid, 'time', nf90_double, dim_time, id_time)
```

Comment:
should it be a double or an integer?

Reply:
@CoryMartin-NOAA, when I worked with @barlage to process all land data, we use double for time. I am not sure if I should use integer. @barlage, Mike, do you have any suggestion to Cory's question?